lua/lazyvim/plugins/extras/lang/rust.lua

```diff
@@ -79,6 +99,14 @@ return {
       },
     },
     config = function(_, opts)
+      -- Configure dap with codelldb
+      local package_path = require("mason-registry").get_package("codelldb"):get_install_path()
+      opts.dap = {
+        adapter = require("rustaceanvim.config").get_codelldb_adapter(
+          package_path .. "/codelldb",
+          package_path .. "/extension/lldb/lib/liblldb.dylib"
```

Comment:
Mmmm, now that I relook at this, it requires some changes to also work on Linux..